### PR TITLE
added glam related notes to integrating-glean-for-product-managers.md

### PR DESCRIPTION
**Pull request by alekhyamoz**
The PR related to the glam [issue](https://github.com/mozilla/glam/issues/1550)
Added notes detailing next steps required to onboard Glean apps to GLAM

### CODE REVIEW

**Review by travis79 — APPROVED** [review]
Thanks for doing this!

File: docs/user/user/integrating-glean-for-product-managers.md [hunk 1]
```diff
@@ -43,3 +43,12 @@ Occasionally, products need to collect data that doesn't fit neatly into one of
 Glean [has a process](https://wiki.mozilla.org/Glean/Adding_or_changing_Glean_metric_types) to request and introduce more metric types and we will work with you to design something appropriate.
 This design and implementation work is at least 4 weeks, though we are working on the foundation to accelerate that.
 Having a telemetry plan (see above) will help to identify this work early.
+
+## Integrating Glean into GLAM
+
+To use GLAM for analysis of your application's data [file a ticket in the GLAM repository](https://github.com/mozilla/glam/issues/new/choose) with the following information:
```
Comment (badboy) [inline docs/user/user/integrating-glean-for-product-managers.md:49]: Can we add a template to the GLAM repository?
That is: copy https://github.com/mozilla/glam/blob/main/.github/ISSUE_TEMPLATE/bug_report.md to something like `add_to_glam.md` and have the teplate ask for that info?

File: docs/user/user/integrating-glean-for-product-managers.md [hunk 2]
```diff
@@ -43,3 +43,12 @@ Occasionally, products need to collect data that doesn't fit neatly into one of
...
+
+1. Application Id
+2. Dataset location
+
+A data engineer from GLAM team will reach out to you if further information is required.
```
Comment (badboy) [inline docs/user/user/integrating-glean-for-product-managers.md:54]:
```suggestion
A data engineer from the GLAM team will reach out to you if further information is required.
```

**Review by badboy — CHANGES_REQUESTED** [review]

File: docs/user/user/integrating-glean-for-product-managers.md [hunk 3]
```diff
@@ -43,3 +43,12 @@ Occasionally, products need to collect data that doesn't fit neatly into one of
 Glean [has a process](https://wiki.mozilla.org/Glean/Adding_or_changing_Glean_metric_types) to request and introduce more metric types and we will work with you to design something appropriate.
 This design and implementation work is at least 4 weeks, though we are working on the foundation to accelerate that.
 Having a telemetry plan (see above) will help to identify this work early.
+
+## Integrating Glean into GLAM
+
+To use GLAM for analysis of your application's data [file a ticket in the GLAM repository](https://github.com/mozilla/glam/issues/new/choose) with the following information:
+
+1. Application Id
+2. Dataset location
```
[the remark this answers is not in the document]
Comment (alekhyamoz) [reply docs/user/user/integrating-glean-for-product-managers.md]: Removed the section. As the information is being requested in the template

**Review by badboy — APPROVED** [review]

**Pull request merged**